hpedockerplugin/hpe_storage_api.py

```diff
@@ -334,10 +334,15 @@ def volumedriver_create(self, name, opts=None):
                         return response
 
             rcg_name = contents['Opts'].get('replicationGroup', None)
```

Comment:
Except for the comment addition  I don't see any change here. Anything I'm missing here ?

Reply:
validate rcg will now be invoked even if Opts is not present in contents - notice that validate_rcg call is not part of the if condition it used to be earlier.

Reply:
@imran-ansari , thanks for the clarification, I did'nt notice the tab.